content/docs/setup/customize/aws-elb/alb.md

```diff
@@ -8,9 +8,13 @@ In this guide you explore how to expose the {{< reuse "docs/snippets/product-nam
 {{< callout type="warning" >}}
 Keep in mind the following considerations when working with an ALB: 
 * The AWS Load Balancer Controller only supports creation of an ALB through an Ingress Controller and not through the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}}. Because of this, you must create the ALB separately and later connect it to the service that exposes your gateway proxy.
+* Currently, {{< reuse "docs/snippets/product-name.md" >}} does not support support enabling health checks on the gateway. The default AWS ELB health checks are automatically created when you create the Gateway resource and might report that the gateway proxy is unhealthy, even if it is healthy.
```

Comment:
Summary for testing the ALB guide:
- Healthcheck in HTTPListenerPolicy is not yet supported, so I commented out steps for configuring that healthcheck throughout.
- You can still follow the rest of this guide just fine to create the ALB, but it'll show instances as unhealthy.
- I added this note here and later in the steps calling this out.

What do we think? Should this guide be published regardless of this issue, or should we continue to hide it until configuring custom healthchecks becomes supported?

Reply:
without successful health checks, the ALB is not usable so I think we should not publish this guide. But I think we should make Lawrence aware of this issue so that they can decide what to do with it

Reply:
hmm interesting. in the past, I was not able to use the ALB when the health checks were failing, but maybe that is because it was actually broken whereas in your case,  it is not broken, it just does not look nice

Reply:
maybe let's ask about it in the kgateway-dev channel to see what we should do with it